### ISSUE TITLE
Improve stereo card detection

### ISSUE THREAD

**Issue by drtrigon** [AbdealiJK/file-metadata#61]
Mental note (stumbled upon this when googling ;) for when going back to IT8 detection (working on that) again:
What about splitting the image part (not the scale) in left and right and use http://stackoverflow.com/questions/11541154/checking-images-for-similarity-with-opencv to compare?

**Comment by AbdealiLoKo** [comment]
This would be for stereo card detection rather than IT8 Strip detection right ?

**Comment by drtrigon** [comment]
Yes, sorry - I was having to wrong thing in mind! Adopted the caption.

Can we use the color bar/scale detection from IT8 (see https://github.com/pywikibot-catfiles/file-metadata/issues/62) for the stereo cards in the https://commons.wikimedia.org/wiki/Category:NYPL_stereoscopic_views set also?

**Comment by AbdealiLoKo** [comment]
The IT8 detection and Stereo cards are independent - IT8 can be for things that are not stereo cards, and stereo cards need not always have IT8 strips.

So, yeah both Stereo card detection and IT8 Detection would be applicable for NYPL_stereoscopic_views
